argus/action_manager/windows.py

```diff
@@ -53,10 +53,36 @@ class WindowsActionManager(base.BaseActionManager):
 
     WINDOWS_MANAGEMENT_CMDLET = "Get-WmiObject"
     _INSTALL_SCRIPT = r"C:\installCBinit.ps1"
+    _ENCODED_SCRIPT = r"C:\encode.py"
```

Comment:
For the future development I think a better idea will be to put there a dispatcher.